### PR TITLE
Btrfs-progs: bump to 5.6 and depends change

### PR DESCRIPTION
**Pull request by aicsx**
## Description of package
Btrfs is a copy on write (COW) filesystem for Linux aimed at implementing advanced features while focusing on fault tolerance, repair and easy administration.

## Installed manifest of package

(`cat  /var/db/kiss/installed/btrfs-progs/manifest`)

```
/var/db/kiss/installed/btrfs-progs/version
/var/db/kiss/installed/btrfs-progs/sources
/var/db/kiss/installed/btrfs-progs/manifest
/var/db/kiss/installed/btrfs-progs/depends
/var/db/kiss/installed/btrfs-progs/checksums
/var/db/kiss/installed/btrfs-progs/build
/var/db/kiss/installed/btrfs-progs/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/lib/udev/rules.d/64-btrfs-dm.rules
/usr/lib/udev/rules.d/
/usr/lib/udev/
/usr/lib/libbtrfsutil.so.1.2.0
/usr/lib/libbtrfsutil.so.1
/usr/lib/libbtrfsutil.so
/usr/lib/libbtrfsutil.a
/usr/lib/libbtrfs.so.0.1
/usr/lib/libbtrfs.so.0
/usr/lib/libbtrfs.so
/usr/lib/libbtrfs.a
/usr/lib/
/usr/include/btrfsutil.h
/usr/include/btrfs/version.h
/usr/include/btrfs/sizes.h
/usr/include/btrfs/send.h
/usr/include/btrfs/send-utils.h
/usr/include/btrfs/send-stream.h
/usr/include/btrfs/rbtree.h
/usr/include/btrfs/raid56.h
/usr/include/btrfs/radix-tree.h
/usr/include/btrfs/list.h
/usr/include/btrfs/kerncompat.h
/usr/include/btrfs/ioctl.h
/usr/include/btrfs/extent_io.h
/usr/include/btrfs/extent-cache.h
/usr/include/btrfs/ctree.h
/usr/include/btrfs/crc32c.h
/usr/include/btrfs/btrfsck.h
/usr/include/btrfs/btrfs-list.h
/usr/include/btrfs/
/usr/include/
/usr/bin/mkfs.btrfs
/usr/bin/fsck.btrfs
/usr/bin/btrfstune
/usr/bin/btrfsck
/usr/bin/btrfs-select-super
/usr/bin/btrfs-map-logical
/usr/bin/btrfs-image
/usr/bin/btrfs-find-root
/usr/bin/btrfs-convert
/usr/bin/btrfs
/usr/bin/
/usr/
```

## Existing package

- [x] I am the maintainer of this package.

### CODE REVIEW

File: community/btrfs-progs/depends [hunk 1]
```diff
@@ -1,2 +1,4 @@
 lzo
 python
+util-linux
+zlib
```
Comment (dylanaraps) [inline community/btrfs-progs/depends:4]: Missing a newline here.

File: community/btrfs-progs/sources [hunk 2]
```diff
@@ -1 +1 @@
-https://www.kernel.org/pub/linux/kernel/people/kdave/btrfs-progs/btrfs-progs-v5.4.1.tar.xz
+https://www.kernel.org/pub/linux/kernel/people/kdave/btrfs-progs/btrfs-progs-v5.6.tar.xz
```
Comment (dylanaraps) [inline community/btrfs-progs/sources:1]: Missing a newline here.

File: community/btrfs-progs/version [hunk 3]
```diff
@@ -1 +1 @@
-5.4.1 1
+5.6 1
```
Comment (dylanaraps) [inline community/btrfs-progs/version:1]: Missing a newline here.

Comment (aicsx) [reply community/btrfs-progs/version]: yeha sorry, fixed.

**Pull request merged**